kodi_addon_checker/xml_schema/gotham_metadata.xsd

```diff
@@ -8,14 +8,14 @@
         <xs:element name="description" type="translatedString" minOccurs="0" maxOccurs="unbounded"/>
         <xs:element name="disclaimer" type="translatedString" minOccurs="0" maxOccurs="unbounded"/>
         <xs:element name="platform" type="platformList" minOccurs="0" maxOccurs="1"/>
-        <xs:element name="language" type="nonEmptyString"  minOccurs="0" maxOccurs="1"/>
+        <xs:element name="language" type="xs:string"  minOccurs="0" maxOccurs="1"/>
```

Comment:
I'd leave language as a `nonEmptyString `. It is often a source of confusion between addon submitters and if included we should force users to define it. We can always edit the wiki

Reply:
Can you please go ahead do that and edit the wiki? I will change the PR then.

Reply:
So what's the difference to allow empty forum, email and news, but not language.
Either allow all to be empty, which in my opinion gives no additional info and bloats the addon.xml, or require all to be non empty or not added.

Reply:
done

Reply:
language must be allowed empty.

Reply:
@Rechi I think language (unlike the others) is the only used in the Kodi GUI...likely ignored if empty though. I agree with you that empty tags only bloat the addon.xml but suddenly blocking user PRs is not a good alternative either. In the end, if Kodi does not complain it is up to the developer to have such a verbose manifest. We can warn during reviews if we spot such stuff